### PR TITLE
Replace c1

### PR DESCRIPTION
**Pull request by dczulada**

### CODE REVIEW

**Comment by dczulada** [comment]
Original C1 Testing

before

Edit Test to replace Encounter, Performed with Medication, Allergy

edit

Updated Test

after

File: app/helpers/checklist_tests_helper.rb [hunk 1]
```diff
@@ -27,6 +27,35 @@ def fulfills_reference(measure, referenced_criteria)
     end
   end
 
+  def availble_data_criteria(measure, criteria)
```
Comment (okeefm) [inline app/helpers/checklist_tests_helper.rb]: `availble` should be `available`

File: app/views/checklist_tests/_checklist_measure.html.erb [hunk 2]
```diff
@@ -107,6 +105,9 @@
                           <% else %>
                             <td/>
                           <% end %>
+                          <td class='show-me' style='display: none;'>
+                            <%= criteria_field.select :replacement_data_criteria, availble_data_criteria(measure, criteria), :selected => criteria_field.object.source_data_criteria, class: 'show-me' %>
```
Comment (okeefm) [inline app/views/checklist_tests/_checklist_measure.html.erb]: `availble` should be changed to `available` here as well

File: app/helpers/checklist_tests_helper.rb (hunk 1 above)
Comment (dczulada) [inline app/helpers/checklist_tests_helper.rb]: Another example of where going to the Derek Zoolander center for kids who can't read good really isn't paying off.

**Comment by okeefm** [comment]
The only last comment I have here is that it's not obvious to me what the "Toggle Negation" switch does. Does that toggle the negation of the entire valueset or just a single code? If so, we may want to grey out the code selector when the toggle is on, or change the text that says "negate entire valueset", so it's more obvious what the switch is doing.

**Comment by dczulada** [comment]
If I click the toggle now, it will negate the entire valueset
valueset

If I click the toggle now, it will negate an individual code
single

**Pull request merged**